tdrs-backend/tdpservice/parsers/test/test_util.py

```diff
@@ -56,11 +64,11 @@ def test_parse_line_parses_line_from_schema_to_dict():
     schema = RowSchema(
         model=dict,
         fields=[
-            Field(item=1, name='first', type='string', startIndex=0, endIndex=3),
-            Field(item=2, name='second', type='string', startIndex=3, endIndex=4),
-            Field(item=3, name='third', type='string', startIndex=4, endIndex=5),
-            Field(item=4, name='fourth', type='number', startIndex=5, endIndex=7),
-            Field(item=5, name='fifth', type='number', startIndex=7, endIndex=8),
+            Field(item=1, name='first', friendly_name='first', type='string', startIndex=0, endIndex=3),
```

Comment:
formatting for whole file reverted from PR

Reply:
not sure what you mean by standardized format here. 
this is a test file, so formatting doesn't matter as it is only for functional testing.